wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/UpdateConversationProtocolActionHandlerTests.swift

```diff
@@ -0,0 +1,124 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import WireRequestStrategy
+
+final class UpdateConversationProtocolActionHandlerTests: ActionHandlerTestBase<UpdateConversationProtocolAction, UpdateConversationProtocolActionHandler> {
+
+    override func setUp() {
+        super.setUp()
+
+        let uuid = UUID(uuidString: "b906579d-60dd-4510-a3ca-14b2ec225f4a")!
```

Comment:
use constant that can be reuse in the assertion of the path

Reply:
done: d362da01065a1abfa71436d2bb754088a680e40c